core/services/flux_monitor.go

```diff
@@ -206,24 +209,28 @@ func (fm *concreteFluxMonitor) RemoveJob(ID *models.ID) {
 // DeviationCheckerFactory holds the New method needed to create a new instance
 // of a DeviationChecker.
 type DeviationCheckerFactory interface {
-	New(models.Initiator, RunManager) (DeviationChecker, error)
+	New(models.Initiator, RunManager, *orm.ORM) (DeviationChecker, error)
 }
 
 type pollingDeviationCheckerFactory struct{}
 
-func (f pollingDeviationCheckerFactory) New(initr models.Initiator, runManager RunManager) (DeviationChecker, error) {
+func (f pollingDeviationCheckerFactory) New(initr models.Initiator, runManager RunManager, orm *orm.ORM) (DeviationChecker, error) {
 	if initr.InitiatorParams.PollingInterval < MinimumPollingInterval {
 		return nil, fmt.Errorf(
 			"pollingInterval must be equal or greater than %s",
 			MinimumPollingInterval,
 		)
 	}
 
+	urls, err := ExtractFeedURLs(initr.InitiatorParams.Feeds, orm)
+	if err != nil {
+		return nil, err
+	}
+
 	fetcher, err := newMedianFetcherFromURLs(
 		defaultHTTPTimeout,
 		initr.InitiatorParams.RequestData.String(),
-		initr.InitiatorParams.Feeds...)
-
+		urls)
```

Comment:
I believe the business case may have introduced a more complex scenario now.

@se3000  How is this kept in sync with the life cycle of a Bridge?  What happens if it's URL is updated or deleted?  The deleted case is addressed in the task, but doesn't seem to be in the PR.
```
Validate the bridge name when the job is created. If the bridge is no longer present at runtime, error out.
```

Reply:
hmmmm really good point here... seems like a really easy for the FM to get out of wack if bridges get renamed / deleted.

Reply:
We've talked about validating this on delete. I think there's another story buried in the icebox, but here's another in case not: https://www.pivotaltracker.com/story/show/171217062

Reply:
I've noted the problem of updating the bridge name there.